html/semantics/interactive-elements/the-dialog-element/dialog-form-submission-unusual.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<title>Test dialog form submission, unusual cases</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<div id=log></div>
+
+<dialog>
+  <form method=dialog action="https://test:test/" rel=noopener>
+    <input type=submit formaction="https://test:test/">
+    <input type=submit>
+  </form>
+</dialog>
+
+<script>
+test(() => {
+  const dialog = document.querySelector("dialog");
+  dialog.showModal();
+  assert_true(dialog.open);
+
+  document.querySelector("input").click();
```

Comment:
This test and the one below would be much easier to understand (and less fragile to refactoring) if the two submit buttons had separate IDs.